src/averageAges.js

```diff
@@ -37,7 +49,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let women = people.filter(({ sex }) => sex === SEX_FEMALE);
+
+  const mothers = women.filter(({ name }) =>
```

Comment:
You don't need calculate this if `withChildren` is false. Move it inside `if` block